CHANGES.txt

```diff
@@ -5,6 +5,8 @@ Changes for Crate
 Unreleased
 ==========
 
+ - Added support for subselect statements without nested aggregations.
```

Comment:
I don't think this PR adds the functionality that this changes entry would imply